_opcodes/op453-iwd2.html

```diff
@@ -10,4 +10,10 @@
 iwd2: 1
 pst: 0
 ---
-Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages. This effect is used internally.
+Disables the usage of Expertise, detects the activation/revocation of Power Attack, prints appropriate messages.
+
+{% capture note %}
+This effect is used internally.
```

Comment:
My thought was, if somebody want's to use these, they should clearly see, that these are internal to the engine and maybe it's better to look for an alternative